### ISSUE TITLE
Black Box or Console Blinking When Using os.execute and command line

### ISSUE THREAD

**Issue by Wajinn** [vlang/v#13706]
Would like to see an option with os.execute that completely hides the console and prevents any kind of blinking or pop-up.  Sometimes, for various reasons, the command prompt console will show for a fraction of a second (blink).  

I know that this is possible, as have seen it in other languages, where they have a cross-platform Process or RunCommand that has a "NoConsole" option.  Would be great if V has this too.
```
import os

fn main() {
	println(os.execute(r'cmd /c ipconfig /all').noconsole())  // maybe something like this or whatever is best
}
```

**Comment by spytheman** [comment]
try just `os.execute('ipconfig /all')`

**Comment by Wajinn** [comment]
> try just `os.execute('ipconfig /all')`

Yes, have typed it that way and other ways.  It still does quick fraction of the second pop-ups on Window machines.  Particularly when first ran or hasn't been run for some length of time.  Would be great to have some kind of "no_console" option.

**Comment by Dialga** [comment]
You may need to pass the -mwindows option to gcc to hide the command prompt, I haven't tried this.
https://stackoverflow.com/a/622602/1461607
